### ISSUE TITLE
helm-match-plugin has been renamed

### ISSUE THREAD

**Issue by fletch** [yasuyk/helm-company#20]
Upgraded to helm-20150824.449 and helm-company stopped loading.  I don't see anything offhand in helm-company using anything defined in the helm-match-plugin but the file was renamed and the corresponding functions got renamed `s/-mp-/-mm-/`.

Still poking but just the following diff got things loading. 

    diff --git a/helm-company.el b/helm-company.el
    index 580391d..9de7714 100644
    --- a/helm-company.el
    +++ b/helm-company.el
    @@ -33,7 +33,7 @@
     ;;; Code:
    
     (require 'helm)
    -(require 'helm-match-plugin)
    +(require 'helm-multi-match)
     (require 'helm-files)
     (require 'helm-elisp) ;; For with-helm-show-completion
     (require 'company)

**Comment by manuel-uberti** [comment]
I can confirm this is an issue. I get this error when trying to use helm-company in Emacs shell (```M-x shell```):

```
Debugger entered--Lisp error: (file-error "Cannot open load file" "No such file or directory" "helm-match-plugin")
  require(helm-match-plugin)
  byte-code("\300\301!\210\300\302!\210\300\303!\210\300\304!\210\300\305!\210\306\307\310\311\312\313\314\301&\210\315\316\317\320\314\307\321\322&\207" [require helm helm-match-plugin helm-files helm-elisp company custom-declare-group helm-company nil "Helm interface for company-mode." :prefix "helm-company-" :group custom-declare-variable helm-company-candidate-number-limit 300 "Limit candidate number of `helm-company'.\n\nSet it to nil if you don't want this limit." :type (choice (const :tag "Disabled" nil) integer)] 8)
  call-interactively(helm-company)
  yas--fallback()
  yas-expand()
  funcall-interactively(yas-expand)
  call-interactively(yas-expand nil nil)
  command-execute(yas-expand)
```

**Comment by yasuyk** [comment]
Sorry for the very very delayed response.

This issue is fixed at 3b59a23.
